step-strikers-app/step-strikers-app/LoginViewController.swift

```diff
@@ -34,8 +40,65 @@ class LoginViewController: UIViewController {
         signupButton.addTarget(self, action:#selector(signupPressed), for:.touchUpInside)
     }
 
+    // when testing this use username: jazzyjalyn, password: doesn't matter yet but can't be empty
     @objc func loginPressed(_ sender: Any) {
-        // TODO: Validate user credentials and navigate to the Stats screen
+        if self.usernameTextField!.text == "" {
+            // TODO: @Nick don't print this, display it on the screen
+            print("username not entered")
+        } else if passwordTextField!.text == "" {
+            // TODO: @Nick don't print this, display it on the screen
+            print("password not entered")
+        } else {
+            // TODO: @Kelly validate password
+            let docRef = Firestore.firestore().collection("players").document(self.usernameTextField!.text!)
+            docRef.getDocument { (document, error) in if let document = document, document.exists {
+                // TODO: @jalyn you can remove all these prints after you're done. I used them to test and left them in case they're helpful!
+                let characterName = document.get("character_name") as! String
```

Comment:
Two things:
Does firebase store the class of the character (e.g bard, fights, etc.) If so, I would need this.
Also need current spell points for the casters.